ocp4/profiles/coreos-ncp.profile

```diff
@@ -140,75 +133,75 @@ selections:
 
     ### Kernel Config
     ## Boot prompt
-    #- grub2_audit_argument
-    #- grub2_audit_backlog_limit_argument
-    #- grub2_slub_debug_argument
-    #- grub2_page_poison_argument
-    #- grub2_vsyscall_argument
-    #- grub2_vsyscall_argument.role=unscored
-    #- grub2_vsyscall_argument.severity=info
-    #- grub2_pti_argument
+    - grub2_audit_argument
+    - grub2_audit_backlog_limit_argument
+    - grub2_slub_debug_argument
+    - grub2_page_poison_argument
+    - grub2_vsyscall_argument
+    - grub2_vsyscall_argument.role=unscored
+    - grub2_vsyscall_argument.severity=info
+    - grub2_pti_argument
 
     ## Security Settings
-    #- sysctl_kernel_kptr_restrict
-    #- sysctl_kernel_dmesg_restrict
-    #- sysctl_kernel_kexec_load_disabled
-    #- sysctl_kernel_yama_ptrace_scope
-    #- sysctl_kernel_perf_event_paranoid
-    #- sysctl_user_max_user_namespaces
-    #- sysctl_user_max_user_namespaces.role=unscored
-    #- sysctl_user_max_user_namespaces.severity=info
-    #- sysctl_kernel_unprivileged_bpf_disabled
-    #- sysctl_net_core_bpf_jit_harden
+    - sysctl_kernel_kptr_restrict
+    - sysctl_kernel_dmesg_restrict
+    - sysctl_kernel_kexec_load_disabled
+    - sysctl_kernel_yama_ptrace_scope
+    - sysctl_kernel_perf_event_paranoid
+    - sysctl_user_max_user_namespaces
+    - sysctl_user_max_user_namespaces.role=unscored
+    - sysctl_user_max_user_namespaces.severity=info
+    - sysctl_kernel_unprivileged_bpf_disabled
+    - sysctl_net_core_bpf_jit_harden
 
     ## File System Settings
-    #- sysctl_fs_protected_hardlinks
-    #- sysctl_fs_protected_symlinks
+    - sysctl_fs_protected_hardlinks
+    - sysctl_fs_protected_symlinks
 
     ### Audit
-    #- var_auditd_flush=incremental_async
-    #- auditd_data_retention_flush
-    #- auditd_local_events
-    #- auditd_write_logs
-    #- auditd_log_format
-    #- auditd_freq
-    #- auditd_name_format
-    #- var_auditd_action_mail_acct=root
-    #- var_auditd_space_left_action=email
-    #- auditd_audispd_configure_remote_server
-    #- auditd_audispd_encrypt_sent_records
-    #- auditd_audispd_disk_full_action
-    #- auditd_audispd_network_failure_action
-
+    - service_auditd_enabled
+    - var_auditd_flush=incremental_async
+    - auditd_data_retention_flush
+    - auditd_local_events
+    - auditd_write_logs
+    - auditd_log_format
+    - auditd_freq
+    - auditd_name_format
+    - var_auditd_action_mail_acct=root
```

Comment:
Is it reasonable to expect that root will read their mail on a cluster node?

Reply:
Good question! Generally, root email is aliased/forwarded to a distribution group (e.g. cluster-admins@customer.com). For the Government baselines this is expected and accepted. But something to revisit for the commercial profiles.